wiki/en/en-Setup.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: wiki
+title: "Setup"
+lang: "en"
+permalink: "/wiki/Setup"
+---
+
+# Setup
+
+To get the best from Jamulus, at a minimum you will need:
+
+1. **A wired internet connection** (not wi-fi - see the [FAQ](/wiki/FAQ))
+1. **Wired headphones** (not Bluetooth or speakers - see the [FAQ](/wiki/FAQ))
+1. **A reasonable audio device, sound card and/or mic** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) for examples)
+
+If you have any questions, please see the [Discussions](https://github.com/jamulussoftware/jamulus/discussions)
+## Getting started
+If you haven’t done so already, please follow the installation guide for your platform to get Jamulus working with your hardware:
+
+* [Windows](/wiki/Installation-for-Windows)
```

Comment:
... which might hint at the need of another rewrite of the windows page and maybe a separate page for ASIO4ALL?

Reply:
I wrote this page so that it would fit in reasonably well with the installer pages (and removed all the overlapping/duplicated stuff). Was there anything in those pages that needs changing? If so can add that to this PR.

Reply:
> There is this we can link to. In fact maybe we should.

Yes, but that's only one part (since it gives specific examples but not a generic help). I think we might leave it as is at the moment but keep it in mind until others come up with an idea. Yes, the ASIO4ALL stuff does make the Windows page look difficult. But to be honest, a lot of windows users do benefit from a detailed ASIO4ALL set up guide.